### PR TITLE
encrypt.rb

### PR DESCRIPTION
**Pull request by agpiermarini**
resolved hound errors

### CODE REVIEW

File: lib/encrypt.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'Date'
```
[the remark this answers is not in the document]
Comment (EMSwank) [reply lib/encrypt.rb]: Don't know what this means.

File: lib/encrypt.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+require 'Time'
+require 'pry'
+
+class Encrypt
```
[the remark this answers is not in the document]
Comment (EMSwank) [reply lib/encrypt.rb]: ?

File: lib/encrypt.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  def random_key(length, ceiling)
+    length.times.map { Random.rand(ceiling) }
+  end
+
+  def key_offset(key = random_key(5, 10))             #problematic for edge cases? can we hard code in random_key and still have appropriate tests
```
[the remark this answers is not in the document]
Comment (EMSwank) [reply lib/encrypt.rb]: Deal with this later
